orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/DestroyServerGroupStage.groovy

```diff
@@ -47,7 +47,7 @@ class DestroyServerGroupStage extends TargetServerGroupLinearStageSupport {
       [
         buildStep(stage, "disableServerGroup", DisableServerGroupTask),
         buildStep(stage, "monitorServerGroup", MonitorKatoTask),
-        buildStep(stage, "waitForDownInstances", WaitForAllInstancesDownTask),
+        buildStep(stage, "waitForNotUpInstances", WaitForAllInstancesNotUpTask),
```

Comment:
We sort of talked about this @anotherchrisberry.

I'm not sure we can do much else to handle disabling of server groups that are in an obviously bad state (ie. stuck in a `STARTING` state).